landscape/how-to/install-landscape-server-on-microsoft-azure.md

```diff
@@ -0,0 +1,205 @@
+
+This guide provides an example of how to install and set up your Landscape server on Microsoft Azure with [cloud-init](https://cloudinit.readthedocs.io/en/latest/). The instructions here can be used for both FIPS-hardened or non-hardened systems.
+
+> **For the most up-to-date documentation on Microsoft Azure, see [Microsoft Azure documentation](https://learn.microsoft.com/en-us/azure).
+
+**Contents:**
+
+- Install and set up Microsoft Azure CLI
+- Provision Azure resources and deploy
+- Deploy Landscape Server VM with cloud-init
+- Configure Landscape
+- (Optional) Perform a complete teardown
+
+
+## Install and set up Microsoft Azure CLI
+
+### Install `Azure CLI`
+
+To install `Azure CLI`, refer to the [Install Azure CLI on Ubuntu](https://documentation.ubuntu.com/azure/en/latest/azure-how-to/instances/install-azure-cli/) guide.
+
+### Connect `Azure` with your Microsoft Azure account
+
+The Azure CLI's default authentication method for logins uses a web browser and access token to sign in.
+
+Run the `az login` command:
+```
+az login
+```
+
+If the Azure CLI can open your default browser, it initiates authorisation code flow and opens the default browser to load an Azure sign-in page.
+
+Sign in with your account credentials in the browser.
+
+
+## Provision Azure resources and deploy
+
+### Create a resource group
+
+Create a resource group to contain all the Azure resources for deploying the VM.  The following command creates a resource group named `Landscape-rg` in the `eastus` location:
+```
+az group create --name Landscape-rg --location eastus
+```
+
+Output will be displayed in JSON format:
+```
+{
+  "id": "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/Landscape-rg",
+  "location": "eastus",
+  "managedBy": null,
+  "name": "Landscape-rg",
+  "properties": {
+    "provisioningState": "Succeeded"
+  },
+  "tags": null,
+  "type": "Microsoft.Resources/resourceGroups"
+}
+```
+
+### Create public-ip address resource
+
+Create static IP address in the `Landscape-rg` resource group:
+```
+az network public-ip create \
+	--resource-group Landscape-rg \
+    --name LandscapePublicIP \
+    --location eastus \
+    --allocation-method Static
+```
+
+Output will be displayed in JSON format and will show static IP address (extract below):
+```
+"ipAddress": "34.139.255.120",
+"ipTags": [],
+"location": "eastus",
+"name": "LandscapePublicIP",
+"provisioningState": "Succeeded",
+"publicIPAddressVersion": "IPv4",
+"publicIPAllocationMethod": "Static",
+"resourceGroup": "Landscape-rg",
+```
+
+Copy the IP address and set it as the A record value for the domain or subdomain that will serve as the FQDN. You set the A record in your DNS service. You can also use Azure DNS zone to host your DNS domain and manage your DNS records.
+
+Verify the A record using `nslookup`. Replace `{landscape.domain.com}` with your FQDN:
+
+```
+nslookup {landscape.domain.com}
+```
+
+You’ll receive output similar to:
+```
+Server:		127.0.0.53
```

Comment:
```suggestion
Server:		127.X.X.XX
```

Reply:
I have left the output from the nslookup command for the dns server and address.  As this is what most users output will be if using Ubuntu, as it points to the loopback address of 127.0.0.53 which systemd-resolved uses.